concordium-contracts-common/src/schema_json.rs

```diff
@@ -1032,7 +1253,7 @@ impl Type {
             Self::Struct(field) => field.to_json_template(),
             Self::ByteList(_) => "<String with lowercase hex>".into(),
             Self::ByteArray(size) => {
-                format!("String of size {size} containing lowercase hex characters.").into()
+                format!("<String of size {size} containing lowercase hex characters.>").into()
```

Comment:
Technically, the string itself will be twice the size because it uses two characters per byte

Reply:
I changed it to `2*size` which seems more intuitive if people check the size of the `string`.